torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -61,12 +61,12 @@ Expr* LoopNestGenerator::pushAlloc(TensorView* tv) {
 
   // Multiply all the dimensions we're going to use for the allocation together
   // to get the total size
-  Val* size;
+  Val* size = nullptr;
```

Comment:
Why bother when we're guaranteed to set it to something below?
If it's triggering, why not:
```
Val* size = alloc_dims.size() == 0 ? new Int(1) : alloc_dims[0];
```

Reply:
1. Consistency. It builds muscle memory, for best effect this should be a pattern where one doesn't have to think about twice.

2. The practice guards no only against current code, but also future modifications.